### PR TITLE
Updated Symbols - New DAO name

### PR DESCRIPTION
**Pull request by juanky201271**
I found the commit with this change. I imagine that it is good to change the denomination of the App.

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -604,4 +610,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 16d9b8743d81976fd242188201758f6cfce4466a
 
-COCOAPODS: 1.10.1
+COCOAPODS: 1.11.2
```
Comment (zancas) [inline ios/Podfile.lock:613]: What's COCOAPODS?

**Comment by zancas** [comment]
Does that look like the correct name to others?   @AloeareV @Gunnar-Stunnar @dannasessha

File: ios/Podfile.lock (hunk 1 above)
Comment (juanky201271) [reply ios/Podfile.lock:613]: As far as I know Cocoapods is something in Mac like yarn/npm in others environments. A package installer...

**Comment by zancas** [comment]
Since no-one has weighed in with a strong counter-opinion...

**Pull request merged**